com.valvesoftware.Steam.Utility.InhibitScreensaver.yml

```diff
@@ -0,0 +1,63 @@
+id: com.valvesoftware.Steam.Utility.InhibitScreensaver
+sdk: org.freedesktop.Sdk//23.08
+runtime: com.valvesoftware.Steam
+runtime-version: stable
+build-extension: true
+appstream-compose: false
+separate-locales: false
+rename-icon: inhibit-screensaver
+rename-desktop-file: inhibit-screensaver.desktop
+command: inhibit-screensaver
```

Comment:
there is no command. it's an extension

Reply:
I've removed the command in [daf11e0](https://github.com/flathub/flathub/pull/4897/commits/daf11e0d88e93101f43d40aa86d6daf89e5452ec).